packages/replay-internal/src/util/createPerformanceEntries.ts

```diff
@@ -43,7 +43,13 @@ export interface Metric {
    * The array may also be empty if the metric value was not based on any
    * entries (e.g. a CLS value of 0 given no layout shifts).
    */
-  entries: PerformanceEntry[] | PerformanceEventTiming[];
```

Comment:
What happened to `PerformanceEventTiming`?

Reply:
I originally added `PerformanceEventTiming` for INP which extends from `PerformanceEntry`, but I realized that we don't need to use any of the types in `PerformanceEventTiming` so I removed it. I think I forgot to remove it in the previous PRs.